README.md

```diff
@@ -23,83 +23,93 @@ v0.9.0 release, see for details: https://github.com/nutsdb/nutsdb/issues/167
-   - [Badger](#badger)
-- [Benchmarks](#benchmarks)
-- [Caveats & Limitations](#caveats--limitations)
-- [Contact](#contact)
-- [Contributing](#contributing)
-- [Acknowledgements](#acknowledgements)
-- [License](#license)
+  - [Getting Started](#getting-started)
+    - [Installing](#installing)
+    - [Opening a database](#opening-a-database)
+    - [Options](#options)
+      - [Default Options](#default-options)
+    - [Transactions](#transactions)
+      - [Read-write transactions](#read-write-transactions)
+      - [Read-only transactions](#read-only-transactions)
+      - [Managing transactions manually](#managing-transactions-manually)
+    - [Using buckets](#using-buckets)
+      - [Iterate buckets](#iterate-buckets)
+      - [Delete bucket](#delete-bucket)
+    - [Using key/value pairs](#using-keyvalue-pairs)
+    - [Using TTL(Time To Live)](#using-ttltime-to-live)
+    - [Iterating over keys](#iterating-over-keys)
+      - [Prefix scans](#prefix-scans)
+      - [Prefix search scans](#prefix-search-scans)
+      - [Range scans](#range-scans)
+      - [Get all](#get-all)
+    - [Merge Operation](#merge-operation)
+    - [Database backup](#database-backup)
+    - [Using in memory mode](#using-in-memory-mode)
+    - [Using other data structures](#using-other-data-structures)
+      - [List](#list)
+        - [RPush](#rpush)
+        - [LPush](#lpush)
+        - [LPop](#lpop)
+        - [LPeek](#lpeek)
+        - [RPop](#rpop)
+        - [RPeek](#rpeek)
+        - [LRange](#lrange)
+        - [LRem](#lrem)
+        - [LSet](#lset)
+        - [Ltrim](#ltrim)
+        - [LSize](#lsize)
+      - [Set](#set)
+        - [SAdd](#sadd)
+        - [SAreMembers](#saremembers)
+        - [SCard](#scard)
+        - [SDiffByOneBucket](#sdiffbyonebucket)
+        - [SDiffByTwoBuckets](#sdiffbytwobuckets)
+        - [SHasKey](#shaskey)
+        - [SIsMember](#sismember)
+        - [SMembers](#smembers)
+        - [SMoveByOneBucket](#smovebyonebucket)
+        - [SMoveByTwoBuckets](#smovebytwobuckets)
+        - [SPop](#spop)
+        - [SRem](#srem)
+        - [SUnionByOneBucket](#sunionbyonebucket)
+        - [SUnionByTwoBuckets](#sunionbytwobuckets)
+      - [Sorted Set](#sorted-set)
+        - [ZAdd](#zadd)
+        - [ZCard](#zcard)
+        - [ZCount](#zcount)
+        - [ZGetByKey](#zgetbykey)
+        - [ZMembers](#zmembers)
+        - [ZPeekMax](#zpeekmax)
+        - [ZPeekMin](#zpeekmin)
+        - [ZPopMax](#zpopmax)
+        - [ZPopMin](#zpopmin)
+        - [ZRangeByRank](#zrangebyrank)
+        - [ZRangeByScore](#zrangebyscore)
+        - [ZRank](#zrank)
+      - [ZRevRank](#zrevrank)
+        - [ZRem](#zrem)
+        - [ZRemRangeByRank](#zremrangebyrank)
+        - [ZScore](#zscore)
+    - [Comparison with other databases](#comparison-with-other-databases)
+      - [BoltDB](#boltdb)
+      - [LevelDB, RocksDB](#leveldb-rocksdb)
+      - [Badger](#badger)
+    - [Benchmarks](#benchmarks)
+  - [Tested kvstore](#tested-kvstore)
```

Comment:
Is the content of these directories redundant？

because there is no corresponding content

Reply:
Same problem from line 99 to line 108

Reply:
Ah man. I think I fixed it now. @xujiajun